projects/1.16/assets/tinkers-construct/tconstruct/book/zh_cn/upgrades/armor/lightspeed.json

```
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:lightspeed_armor",
+  "text": [
+    {
+      "text": "光芒照耀在你的盔甲上，并帮助你跑得更快。"
```

```suggestion
      "text": "光芒照耀在你的靴子上，并帮助你跑得更快。"
```
求证一下，这个东西原文是写的boots，只能加在鞋上吗